src/main.rs

```diff
@@ -348,6 +380,21 @@ impl RustyShare {
         }
     }
 
+    fn register_action(store: Option<SqliteStore>, user: &str, pass: &str) -> Response {
+        if let Some(ref store) = store {
+            let user_id = register_user(&store, user, pass).unwrap();
```

Comment:
Don't call `unwrap` here, it will crash the server. You should `match` on the result and return a `response::internal_server_error()` on error. No need to check that id is `> 0` anyway.

Reply:
OK